### PR TITLE
Create collections article

### PR DESCRIPTION
**Pull request by carlyrevier**
Adds an article for how to create and manage Collections.

Preview link: https://review.learn.microsoft.com/en-us/contribute/content/collections?branch=pr-en-us-818

### CODE REVIEW

**Comment by carlyrevier** [comment]
#sign-off @jehchow

File: Contribute/content/collections.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+When you add content to a collection, it's added to the bottom of the list. You can reorder content by using the **up/down** arrows on the content item, moving it one space at a time. Wait until each individual move interaction completes before selecting the **up/down** arrows again. We're working to improve this experience.
+
+### Delete content
+
+To delete content within a collection, select the **trash can** icon on the content item. A pop-up appears alerting you that the content will be permanently deleted. Select **Yes, delete** to delete the content.
+
+## Add and manage sections
+
+Once you've saved content to your collection, you can add sections to the collection to make it easier to navigate. You can add as many sections as you need. You can also move content items between sections, and you can [rearrange sections](#rearrange-sections) within the collection. And if your organizational needs change, you can [delete sections](#delete-sections) from the collection.
```
Comment (jehchow) [inline Contribute/content/collections.md:100]:
```suggestion
Once you've saved content to your collection, you can add sections to the collection to make it easier to navigate. You can add as many sections as you need. You can also move content items between sections, and you can [rearrange sections](#rearrange-sections) within the collection. And if you need to reorganize, you can [delete sections](#delete-sections) from the collection.
```

**Review by jehchow — COMMENTED** [review]
Looks great! Just a few suggestions :)

**Comment by carlyrevier** [comment]
#sign-off @jehchow 

Rae confirmed this doesn't need to be reviewed by Collections team. Thanks!

**Review by jehchow — APPROVED** [review]

**Pull request merged**